README.md

```diff
@@ -1,7 +1,10 @@
-Chucker
-======
-[![Download](https://api.bintray.com/packages/olivierperez/Chucker/Chucker%3Alibrary/images/download.svg) ](https://bintray.com/olivierperez/Chucker/Chucker%3Alibrary/_latestVersion) [![Build Status](https://travis-ci.org/ChuckerTeam/chucker.svg?branch=master)](https://travis-ci.org/ChuckerTeam/chucker)
+# Chucker
 
+<p align="center">
+  <img src="https://i.imgur.com/GXoIFZ6.png" alt="chucker icon" width="30%"/>
```

Comment:
Why don't use commit the icon in the project, and use it here?

Reply:
Because the icon is going to be merged on `develop` and this CR is against `master`